server.go

```diff
@@ -92,10 +92,12 @@ type Server struct {
 	databases map[string]*database // databases by name
 	shards    map[uint64]*Shard    // shards by id
 	users     map[string]*User     // user by name
+
+	version string // current version of the server
 }
 
 // NewServer returns a new instance of Server.
-func NewServer() *Server {
+func NewServer(version string) *Server {
```

Comment:
We're only doing this so the HTTP responses have the version in there. So we're just pushing into Server, so it can feed it back to the handler. Perhaps we could find some way to provide it directly to the handler code? I mean `server.go` absolutely doesn't need to know the version, yet we're pushing it into its constructor.